### PR TITLE
Translation of the "JS Environment Requirements " page

### PR DESCRIPTION
**Pull request by koka0012**
#1

### CODE REVIEW

File: content/docs/reference-javascript-environment-requirements.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 ---
 
-React 16 depends on the collection types [Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) and [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set). If you support older browsers and devices which may not yet provide these natively (e.g. IE < 11) or which have non-compliant implementations (e.g. IE 11), consider including a global polyfill in your bundled application, such as [core-js](https://github.com/zloirock/core-js) or [babel-polyfill](https://babeljs.io/docs/usage/polyfill/).
+React 16 depende de uma coleção de tipos [Map](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Global_Objects/Map) e [Set](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Global_Objects/Set). Se você quer suportar navegadores antigos e dispositivos que podem não disponibilizar esses tipos nativamente (e.g. IE < 11) ou tenham uma implementação que não esteja nos conformes (e.g. IE 11), considere incluir um polyfill global no bundle de sua aplicação, tal como o pacote [core-js](https://github.com/zloirock/core-js) ou [babel-polyfill](https://babeljs.io/docs/usage/polyfill/).
```
Comment (glaucia86) [inline content/docs/reference-javascript-environment-requirements.md]:
```suggestion
eact 16 depende de uma coleção de tipos [Map](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Global_Objects/Map) e [Set](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Global_Objects/Set). Se você deseje que tenha suportes para navegadores antigos e dispositivos que podem não disponibilizar esses tipos nativamente (e.g. IE < 11) ou tenham uma implementação que não esteja nos conformes (e.g. IE 11), considere incluir um polyfill global no bundle de sua aplicação, tal como o pacote [core-js](https://github.com/zloirock/core-js) ou [babel-polyfill](https://babeljs.io/docs/usage/polyfill/).
```

Comment (koka0012) [reply content/docs/reference-javascript-environment-requirements.md]: commit sent with the changes.

**Comment by koka0012** [comment]
Changes made @glaucia86

File: content/docs/reference-javascript-environment-requirements.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+React 16 depende de uma coleção de tipos [Map](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Global_Objects/Map) e [Set](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Global_Objects/Set). Se você deseje que tenha suportes para navegadores antigos e dispositivos que podem não disponibilizar esses tipos nativamente (e.g. IE < 11) ou tenham uma implementação que não esteja nos conformes (e.g. IE 11), considere incluir um polyfill global no bundle de sua aplicação, tal como o pacote [core-js](https://github.com/zloirock/core-js) ou [babel-polyfill](https://babeljs.io/docs/usage/polyfill/).
```
[the remark this answers is not in the document]
Comment (koka0012) [reply content/docs/reference-javascript-environment-requirements.md]: Better than mine. thx

File: content/docs/reference-javascript-environment-requirements.md [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
 
-A polyfilled environment for React 16 using core-js to support older browsers might look like:
+Um ambiente polyfilled para o React 16 usando core-js para suportar navegadores antigos pode parecer assim:
```
[the remark this answers is not in the document]
Comment (koka0012) [reply content/docs/reference-javascript-environment-requirements.md]: thank you, I couldn't find a word or a way to write this sentence without using polyfilled, but seen you got it.

File: content/docs/reference-javascript-environment-requirements.md [hunk 4]
```diff
@@ -18,13 +18,13 @@ import React from 'react';
...
 ReactDOM.render(
-  <h1>Hello, world!</h1>,
+  <h1>Olá, mundo!</h1>,
   document.getElementById('root')
 );
 ```
 
-React also depends on `requestAnimationFrame` (even in test environments).  
-You can use the [raf](https://www.npmjs.com/package/raf) package to shim `requestAnimationFrame`:
+React também depende da função `requestAnimationFrame` (mesmo em ambiente de teste).
```
[the remark this answers is not in the document]
Comment (koka0012) [reply content/docs/reference-javascript-environment-requirements.md]: Thx, seen to be more natural.

File: content/docs/reference-javascript-environment-requirements.md [hunk 5]
```diff
@@ -18,13 +18,13 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 
 ReactDOM.render(
-  <h1>Hello, world!</h1>,
+  <h1>Olá, mundo!</h1>,
```
Comment (jhonmike) [inline content/docs/reference-javascript-environment-requirements.md]: > ## Glossary and Style Guide
> ...
> * Leave code blocks untranslated. (It is usually OK to translate comments inside them.)

Comment (cezaraugusto) [reply content/docs/reference-javascript-environment-requirements.md]: thanks for the input. I think for this case specifically it's ok to translate strings inside HTML/JSX elements as it doesn't change any structure/logic. I'll update the glossary

**Comment by cezaraugusto** [comment]
cc @glaucia86 I guess all feedback was addressed please merge if it looks ok to you

**Comment by glaucia86** [comment]
@cezaraugusto done! :)

**Pull request merged**